create_topics_response.go

```diff
@@ -74,6 +74,10 @@ func (c *CreateTopicsResponse) headerVersion() int16 {
 	return 0
 }
 
+func (c *CreateTopicsResponse) isValidVersion() bool {
+	return c.Version >= 0 && c.Version <= 2
```

Comment:
I think we support version 3 which simply makes the NumPartitions and ReplicationFactor in the request optional - with -1 which the code would allow.